oracle/project/contracts/oracle-contract/src/interface.sw

```diff
@@ -1,24 +1,37 @@
 library;
 
 abi Oracle {
-    /// Return the owner (node) of the oracle
+    /// Return the owner (node) of the oracle.
     ///
-    /// The owner is initialized to the first deterministically generated wallet using the SDK in Forc.toml
+    /// # Additional Information
+    ///
+    /// The owner is initialized to the first deterministically generated wallet using the SDK in Forc.toml.
```

Comment:
```suggestion
    /// The owner is initialized to the first deterministically generated wallet using the SDK.
```

My bad, configurables have been removed from the manifest a while back but we still use the first value from the SDK.